examples/graphupscale.cpp

```diff
@@ -77,8 +83,7 @@ int main(int argc, char* argv[])
     // vertex_edge and coarse factor
     {
         const auto upscale = GraphUpscale(comm, vertex_edge, coarse_factor,
-                                          spect_tol, max_evects, dual_target,
-                                          scaled_dual, energy_dual, hybridization);
+                                          param);
```

Comment:
very minor issue, this line may join the previous line? Similar issue also for some other files, e.g. `FiniteVolumeMLMC.cpp`.